firewall/privacy_mapper.go

```diff
@@ -833,3 +835,41 @@ func CryptoRandIntn(n int) (int, error) {
 
 	return int(nBig.Int64()), nil
 }
+
+// ObfuscateConfig alters the config string by replacing pubkeys with random
+// values and updates the privacy pair map.
+func ObfuscateConfig(privacyPairs map[string]string, configB []byte) ([]byte,
```

Comment:
very cool! should we also match on channel IDs/outpoints?

Reply:
yes, will add that :+1:!